plexupdate.sh

```diff
@@ -307,6 +307,20 @@ if [ -z "${DISTRO_INSTALL}" ]; then
 			else
 				DISTRO_INSTALL="${REDHAT_INSTALL}"
 			fi
+		elif [ -f /etc/synoinfo.conf ]; then
+		        DISTRO="synology"
+		        if [ "${PUBLIC}" = "yes" ]; then
+			        if [ "${ARCH}" = "x86_64"]; then
```

Comment:
These lines need spaces before the closing brackets, i.e. `[ "${ARCH}" = "x86_64" ]`.

Reply:
i added the spaces